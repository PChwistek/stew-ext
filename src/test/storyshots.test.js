//import initStoryshots from '@storybook/addon-storyshots'

// initStoryshots({ /* configuration options */ }) 
describe('Examining the syntax of Jest tests', () => {
   
  it('sums numbers', () => {
    expect(1 + 2).toEqual(3)
    expect(2 + 2).toEqual(4)
  })

})