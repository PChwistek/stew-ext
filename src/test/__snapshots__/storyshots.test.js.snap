// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button normal 1`] = `
<div
  style={
    Object {
      "padding": "40px",
    }
  }
>
  <div
    className="button button--primary"
    onClick={[Function]}
  >
    <p
      className="button__text"
    >
       
      Test
       
    </p>
  </div>
</div>
`;

exports[`Storyshots Button tertiary 1`] = `
<div
  style={
    Object {
      "padding": "40px",
    }
  }
>
  <div
    className="button button--secondary"
    onClick={[Function]}
  >
    <p
      className="button__text"
    >
       
      Test
       
    </p>
  </div>
</div>
`;

exports[`Storyshots Checkbox checked 1`] = `
<div
  style={
    Object {
      "padding": "40px",
    }
  }
>
  <label
    className="checkbox-container"
  >
    <div
      className="checkbox__small-text"
    >
      Test
    </div>
    <input
      checked={true}
      onChange={[Function]}
      type="checkbox"
    />
    <span
      className="checkmark"
    />
  </label>
</div>
`;

exports[`Storyshots Checkbox unchecked 1`] = `
<div
  style={
    Object {
      "padding": "40px",
    }
  }
>
  <label
    className="checkbox-container"
  >
    <div
      className="checkbox__small-text"
    >
      Test
    </div>
    <input
      checked={false}
      onChange={[Function]}
      type="checkbox"
    />
    <span
      className="checkmark"
    />
  </label>
</div>
`;

exports[`Storyshots Login initial 1`] = `
<div>
  <div
    className="login"
  >
    <div
      className="login__image-container"
    >
      <img
        className="login__image "
        src={Object {}}
      />
    </div>
    <div
      className="login__error"
    >
       
      
       
    </div>
    <div
      className="login__form-row"
    >
      <div
        className="field  undefined && "
      >
        <input
          autoComplete="off"
          autoFocus={true}
          id="usernameField"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyUp={[Function]}
          placeholder="Email"
          type="text"
          value=""
        />
        <label
          className=""
          htmlFor={1}
        >
          Email
        </label>
      </div>
    </div>
    <div
      className="login__form-row"
    >
      <div
        className="field  undefined && "
      >
        <input
          autoComplete="off"
          id="passwordField"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyUp={[Function]}
          placeholder="Password"
          type="password"
          value=""
        />
        <label
          className=""
          htmlFor={1}
        >
          Password
        </label>
      </div>
    </div>
    <div
      className="login__form-row--submit"
    >
      <div
        className="button button--primary"
        onClick={[Function]}
      >
        <p
          className="button__text"
        >
           
          Login
           
        </p>
      </div>
    </div>
    <div
      className="login__form-row--register"
    >
      <div
        className="link"
      >
        Need an account? 
        <a
          href="https://staging.getstew.com/sign-up"
          target="blank"
        >
           Register. 
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Login with username 1`] = `
<div>
  <div
    className="login"
  >
    <div
      className="login__image-container"
    >
      <img
        className="login__image "
        src={Object {}}
      />
    </div>
    <div
      className="login__error"
    >
       
      
       
    </div>
    <div
      className="login__form-row"
    >
      <div
        className="field  undefined && "
      >
        <input
          autoComplete="off"
          autoFocus={true}
          id="usernameField"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyUp={[Function]}
          placeholder="Email"
          type="text"
          value=""
        />
        <label
          className=""
          htmlFor={1}
        >
          Email
        </label>
      </div>
    </div>
    <div
      className="login__form-row"
    >
      <div
        className="field  undefined && "
      >
        <input
          autoComplete="off"
          id="passwordField"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyUp={[Function]}
          placeholder="Password"
          type="password"
          value=""
        />
        <label
          className=""
          htmlFor={1}
        >
          Password
        </label>
      </div>
    </div>
    <div
      className="login__form-row--submit"
    >
      <div
        className="button button--primary"
        onClick={[Function]}
      >
        <p
          className="button__text"
        >
           
          Login
           
        </p>
      </div>
    </div>
    <div
      className="login__form-row--register"
    >
      <div
        className="link"
      >
        Need an account? 
        <a
          href="https://staging.getstew.com/sign-up"
          target="blank"
        >
           Register. 
        </a>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Popup initial 1`] = `
<div
  className="popup-container"
>
  <div
    className="small-screen-warning"
  >
    Please re-size the window! it's too small for stew.
  </div>
  <div
    className="popup"
  >
    <div>
      <div
        className="login"
      >
        <div
          className="login__image-container"
        >
          <img
            className="login__image "
            src={Object {}}
          />
        </div>
        <div
          className="login__error"
        >
           
          
           
        </div>
        <div
          className="login__form-row"
        >
          <div
            className="field  undefined && "
          >
            <input
              autoComplete="off"
              autoFocus={true}
              id="usernameField"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyUp={[Function]}
              placeholder="Email"
              type="text"
              value=""
            />
            <label
              className=""
              htmlFor={1}
            >
              Email
            </label>
          </div>
        </div>
        <div
          className="login__form-row"
        >
          <div
            className="field  undefined && "
          >
            <input
              autoComplete="off"
              id="passwordField"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              onKeyUp={[Function]}
              placeholder="Password"
              type="password"
              value=""
            />
            <label
              className=""
              htmlFor={1}
            >
              Password
            </label>
          </div>
        </div>
        <div
          className="login__form-row--submit"
        >
          <div
            className="button button--primary"
            onClick={[Function]}
          >
            <p
              className="button__text"
            >
               
              Login
               
            </p>
          </div>
        </div>
        <div
          className="login__form-row--register"
        >
          <div
            className="link"
          >
            Need an account? 
            <a
              href="https://staging.getstew.com/sign-up"
              target="blank"
            >
               Register. 
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Textfield empty 1`] = `
<div
  style={
    Object {
      "padding": "40px",
    }
  }
>
  <div
    className="field  undefined && "
  >
    <input
      autoComplete="off"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyUp={[Function]}
      placeholder="Label"
      value=""
    />
    <label
      className=""
      htmlFor={1}
    >
      Label
    </label>
  </div>
</div>
`;

exports[`Storyshots Textfield has error 1`] = `
<div
  style={
    Object {
      "padding": "40px",
    }
  }
>
  <div
    className="field active undefined && error active"
  >
    <input
      autoComplete="off"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyUp={[Function]}
      placeholder="Label"
      value="test"
    />
    <label
      className="error"
      htmlFor={1}
    >
      Some error
    </label>
  </div>
</div>
`;

exports[`Storyshots Textfield some value 1`] = `
<div
  style={
    Object {
      "padding": "40px",
    }
  }
>
  <div
    className="field active undefined && "
  >
    <input
      autoComplete="off"
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      onKeyUp={[Function]}
      placeholder="Label"
      value="test"
    />
    <label
      className=""
      htmlFor={1}
    >
      Label
    </label>
  </div>
</div>
`;
